docs/data-sources/rds_storage_types.md

```diff
@@ -0,0 +1,72 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_storage_types
+
+Use this data source to get the list of RDS storage types.
+
+## Example Usage
+
+```hcl
+variable "instance_id" {}
+
+data "huaweicloud_rds_storage_types" "test" {
+  db_type      = "MySQL"
+  version_name = "8.0"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `db_type` - (Required, String) DB engine. The valid values are **MySQL**, **PostgreSQL**, **SQLServer**.
+
+* `version_name` - (Optional, String) DB version number.
+
+* `ha_mode` - (Optional, String) HA mode. The valid values are **single**, **ha**, **replica**.
```

Comment:
it's better to keep the same parameter names as `data.huaweicloud_rds_flavors`

Reply:
ok, done